libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -261,7 +261,8 @@ export function Tracker(
         trackerConfiguration.useStm ?? true,
         trackerConfiguration.maxLocalStorageQueueSize ?? 1000,
         trackerConfiguration.connectionTimeout ?? 5000,
-        configAnonymousServerTracking
+        configAnonymousServerTracking,
+        !!trackerConfiguration.allowCredentials ?? true
```

Comment:
Ensures even a string value for `allowCredentials` is converted to a `boolean`, in the case of a non-typescript implementation.